INN/umbrella-citylimits
# Pullquote styling

## Changes

This pull request makes the following changes:

- Adds pullquote styling based on the mockups
- Adds support for theme colors in the block editor color picker

Desktop:
![Screen Shot 2019-11-08 at 4 15 14 PM](https://user-images.githubusercontent.com/18353636/68511023-fb927d00-0242-11ea-9bf6-826dd5c3d77b.png)

Mobile:
![Screen Shot 2019-11-08 at 4 15 22 PM](https://user-images.githubusercontent.com/18353636/68511022-fb927d00-0242-11ea-92b3-3ccdae646f73.png)

## Why



For #53

## Testing/Questions

Features that this PR affects:

- Pull quotes


Questions that need to be answered before merging:

- [ ] WP 5.3 isn't out until next week, _maybe_, which includes the ability to modify alignment variations for blocks. Do we want to just hide these via css for now? https://github.com/INN/umbrella-citylimits/issues/53#issuecomment-544551966
- [ ] The quote icons being next to the borders as detailed in the [design](https://user-images.githubusercontent.com/1754187/67215578-b15e6e80-f3ef-11e9-920b-aaeafd3a0093.png) didn't work because `blockquote` wouldn't let me add psuedo elements to it. Any other ideas or is it ok as is?

Steps to test this PR:

1. Add a pull quote to a post
2. Make sure the color picker works
3. Make sure it looks ok on desktop aligned right, left, default
4. Make sure it looks ok on mobile aligned right, left, default
